src/MATLAB.jl

```diff
@@ -53,4 +53,7 @@ function __init__()
     is_windows() && MSession(0)
 end
 
+# deprecations
+nfields(mx::MxArray) = (Base.depwarn("MATLAB.nfields is deprecated, use mxnfields instead.", :nfields), mxnfields(mx))
```

Comment:
I believe you want `;` not `,`. Better to use the multiline form when you have multiple expressions like this.